src/main/resources/fabric.mod.json

```
@@ -24,8 +24,8 @@
     "fabric-screen-api-v1": "*",
     "fabric-key-binding-api-v1": "*",
     "fabric-lifecycle-events-v1": "*",
-    "fabricloader": ">=0.16.10",
-    "minecraft": "~1.21.6"
+    "fabricloader": ">=0.17.2",
+    "minecraft": ">=1.21.9-rc.1 <=1.21.9"
```

I am not sure of the exact minimum version this pull request would support, but I personally believe ranges should be unbounded for maximum versions:

```suggestion
    "minecraft": ">=1.21.9-"
```